### PR TITLE
update readme: neofs has a collection now

### PR DESCRIPTION
**Pull request by 532910**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -79,10 +77,17 @@ Popular application forks with NeoFS support added:
 
 # NeoGo
 
-Take a look at [neo-go](https://github.com/nspcc-dev/neo-go) repository, it's
-all there. [ansible-neogo-n3](https://github.com/nspcc-dev/ansible-neogo-n3)
-can help you running it.
+Take a look at [neo-go](https://github.com/nspcc-dev/neo-go) repository, it's all there.
 
-To learn more about Neo smart contract development in go take a look at [neo-go-sc-wrkshp](https://github.com/nspcc-dev/neo-go-sc-wrkshp).
+To learn more about Neo smart contract development in go take a look at
```
Comment (roman-khimov) [inline profile/README.md:82]: This certainly doesn't need to be changed.

File: profile/README.md [hunk 2]
```diff
@@ -79,10 +77,17 @@ Popular application forks with NeoFS support added:
 
 # NeoGo
 
-Take a look at [neo-go](https://github.com/nspcc-dev/neo-go) repository, it's
-all there. [ansible-neogo-n3](https://github.com/nspcc-dev/ansible-neogo-n3)
-can help you running it.
+Take a look at [neo-go](https://github.com/nspcc-dev/neo-go) repository, it's all there.
```
Comment (roman-khimov) [inline profile/README.md:80]: I'd say that this one can be left as is as well.

File: profile/README.md [hunk 3]
```diff
@@ -79,10 +77,17 @@ Popular application forks with NeoFS support added:
...
 
-To learn more about Neo smart contract development in go take a look at [neo-go-sc-wrkshp](https://github.com/nspcc-dev/neo-go-sc-wrkshp).
+To learn more about Neo smart contract development in go take a look at
+[neo-go-sc-wrkshp](https://github.com/nspcc-dev/neo-go-sc-wrkshp).
 
 There is also a Neo performance testing tool, [neo-bench](https://github.com/nspcc-dev/neo-bench).
+
+
+# Ansible
```
Comment (roman-khimov) [inline profile/README.md:88]: I'm not sure we need a new top-level section for it. The idea behind this document is that we have two main things --- NeoFS (which has a lot of moving parts in it) and NeoGo. All of the associated things should be bound to one of these entities. Likely we can do the same here.

**Review by roman-khimov — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 1 above)
Comment (532910) [reply profile/README.md:82]: It was truncated to for best view.

File: profile/README.md (hunk 2 above)
Comment (532910) [reply profile/README.md:80]: The same here.

File: profile/README.md (hunk 3 above)
Comment (532910) [reply profile/README.md:88]: This document was highly complicated. It's hard to read a lot of repeating words, like ansible ... ansible-role ... managed. So I simplified it, moving all about ansible into separate part.


Anything not clear what you suggest here.

Comment (532910) [reply profile/README.md:88]: Moreover, I believe ansible part should be removed from this page, see #6

File: profile/README.md (hunk 1 above)
Comment (roman-khimov) [reply profile/README.md:82]: It's not related to the change you're actually doing. If it's some whitespace cleanup it should be a separate commit. Or not exist at all.

File: profile/README.md (hunk 2 above)
Comment (roman-khimov) [reply profile/README.md:80]: Ditto.

File: profile/README.md [hunk 4]
```diff
@@ -59,10 +56,11 @@ NeoFS k6 extension for performance testing is in [xk6-neofs](https://github.com/
 
 NeoFS nodes provide only NeoFS API, but there are additional services
 implementing layers above this API, they're called gateways:
- * [neofs-http-gw](https://github.com/nspcc-dev/neofs-http-gw) implements basic operations with a simple HTTP, can be managed with [ansible-neofs-http-gw](https://github.com/nspcc-dev/ansible-neofs-http-gw)
- * [neofs-rest-gw](https://github.com/nspcc-dev/neofs-rest-gw) makes NeoFS API a RESTful one, can be managed with [ansible-neofs-rest-gw](https://github.com/nspcc-dev/ansible-neofs-rest-gw)
- * [neofs-s3-gw](https://github.com/nspcc-dev/neofs-s3-gw) implements a compatible subset of Amazon S3 protocol, can be managed with [ansible-neofs-s3-gw](https://github.com/nspcc-dev/ansible-neofs-s3-gw)
```
Comment (roman-khimov) [inline profile/README.md:59]: BTW, this is also a whitespace change that is not needed.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**